.github/workflows/job_pytorch_models_tests.yml

```diff
@@ -140,11 +140,22 @@ jobs:
           USE_SYSTEM_CACHE: False
           OP_REPORT_FILE: ${{ env.INSTALL_TEST_DIR }}/TEST-torch_unsupported_ops.log
 
+      - name: PagedAttention Test # Good name?
+        run: |
```

Comment:
I suppose you could add a `if: always()` in case any of the previous stages fail and there is still a need to collect the results from, for example, a long nightly run.

Reply:
Good idea, added